scriptmodules/ports/abuse.sh

```diff
@@ -37,6 +38,7 @@ function install_abuse() {
 
 function configure_abuse() {
     mkRomDir "ports"
+    mkRomDir "ports/$md_id"
     moveConfigDir "$home/.abuse" "$md_conf_root/abuse"
-    addPort "$md_id" "abuse" "Abuse" "pushd $md_inst; $md_inst/abuse; popd"
+    addPort "$md_id" "abuse" "Abuse port of run and gum game" "pushd $md_inst; $md_inst/abuse -datadir $romdir/ports/$md_id; popd"
```

Comment:
the description "port of run and gum game" shouldn't be here (it's gun and not gum also).